airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/streams/async_job.py

```diff
@@ -58,6 +58,9 @@ class Status(str, Enum):
 class AsyncJob(ABC):
     """Abstract AsyncJob base class"""
 
+    # max attempts for a job before errroring out
+    max_attempts: int = 10  # TODO: verify a sane number for this
```

Comment:
am I reading this correctly that we currently have zero retry logic for async jobs, and that we just keep splitting until the job fails at the moment? if so this change makes a lot of sense

I'd recommend setting this to something like 5, it seems like if something fails 9 times it's unlikely to succeed on the 10th? (but you never know when it comes to FB :)